libecho/src/AudioOutput.cc

```diff
@@ -1,11 +1,9 @@
 #include "AudioOutput.h"
 
-#include <QtCore/QEventLoop>
-
 
 AudioOutput::AudioOutput(const QAudioFormat &format) : AudioStream(format) {}
 
-void AudioOutput::enqueueData(const char* data, size_t length) {
+void AudioOutput::enqueueData(const char* data, int length) {
```

Comment:
czemu zmieniasz w tę stronę?

Reply:
Bo to po spodem wyowłuje https://doc.qt.io/qt-5/qbytearray.html#append-4 , uznałem że lepiej dostosowac się do tego niż potem castować.